src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -785,5 +788,23 @@ public string GetWinFormButtonPropertyValue(string buttonName, string propertyNa
 
         public void Undo()
             => GetDTE().ExecuteCommand("Edit.Undo");
+
+        /// <summary>
+        /// Gets the spans where a particular tag appears in the active text view.
+        /// </summary>
+        /// <returns>
+        /// Given a list of tag spans [s1, s2, ...], returns a decomposed array for serialization:
+        ///     [s1.Start, s1.Length, s2.Start, s2.Length, ...]
```

Comment:
Please do. We have something similar for signaturehelp and I'm adding something similar for classification.